entrypoint.sh

```diff
@@ -16,6 +16,8 @@ fi
 
 if [[ -n "$INPUT_TFSEC_EXCLUDE" ]]; then
   TFSEC_OUTPUT=$(/go/bin/tfsec ${TFSEC_WORKING_DIR} --no-colour -e "${INPUT_TFSEC_EXCLUDE}" ${INPUT_TFSEC_OUTPUT_FORMAT:+ -f "$INPUT_TFSEC_OUTPUT_FORMAT"} ${INPUT_TFSEC_OUTPUT_FILE:+ --out "$INPUT_TFSEC_OUTPUT_FILE"})
+elif [[ -n "$INPUT_TFSEC_CONFIG_FILE" ]]; then
```

Comment:
@triat thx for getting back. I apologize for the slow response. I'm finally getting a chance to circle back to this. The main reason I used the `elif` was that I followed the same pattern that already existing for `INPUT_TFSEC_EXCLUDE`. Is this how you'd imagine using config file? 

```
if [[ -n "$INPUT_TFSEC_CONFIG_FILE" ]]; then
  TFSEC_CONFIG_FILE="--config-file ${INPUT_TFSEC_CONFIG_FILE}"

if [[ -n "$INPUT_TFSEC_EXCLUDE" ]]; then
  TFSEC_OUTPUT=$(/go/bin/tfsec ${TFSEC_WORKING_DIR} --no-colour ${TFSEC_CONFIG_FILE} -e "${INPUT_TFSEC_EXCLUDE}" ${INPUT_TFSEC_OUTPUT_FORMAT:+ -f "$INPUT_TFSEC_OUTPUT_FORMAT"} ${INPUT_TFSEC_OUTPUT_FILE:+ --out "$INPUT_TFSEC_OUTPUT_FILE"})
else
  TFSEC_OUTPUT=$(/go/bin/tfsec ${TFSEC_WORKING_DIR} --no-colour ${TFSEC_CONFIG_FILE} ${INPUT_TFSEC_OUTPUT_FORMAT:+ -f "$INPUT_TFSEC_OUTPUT_FORMAT"} ${INPUT_TFSEC_OUTPUT_FILE:+ --out "$INPUT_TFSEC_OUTPUT_FILE"})
fi
```

Reply:
At best, I'd like to stay as close as possible to what TFSEC does. And I believe that you should be able to give any parameters without getting another one magically removed (when this makes sense, of course).

This is why I'd prefer to have `if`s that add parameters to the command rather than `if ... else`s that take whatever first parameter you give to your action. Does this make sense?

This might be wrongly done right now, and would require some refactoring if this is wrong. But to be honest, I'm not using it right now and I'm not spending that much time on open source projects recently. This does not mean I won't do it at some point tho.